app/dashboard/models.py

```diff
@@ -2445,7 +2445,7 @@ def get_eth_sum(self, sum_type='collected', network='mainnet', bounties=None):
 
         return eth_sum
 
-    def get_who_works_with(self, work_type='collected', network='mainnet', bounties=None):
+    def get_who_works_with(self, work_type='collected', network='rinkeby', bounties=None):
```

Comment:
shouldn't this be mainnet?

Reply:
Forgot to return this. Used it for testing. I'll fix this now